src/renderer/lib/state.js

```diff
@@ -68,7 +68,7 @@ function getDefaultState () {
 function load (cb) {
   appConfig.read(function (err, saved) {
     if (err) {
-      console.log('Missing config file: Creating new one')
```

Comment:
This message is incorrect, we're not creating the configuration file here, it's missing so we're using default values.